index.src.html

```diff
@@ -534,10 +535,13 @@ <h2 id="algorithms">Algorithms</h2>
 
   <h3 id="parsing">Parsing</h3>
 
-  <h4 id="get-types">
+  <h4 id="algorithm-get-types" algorithm="get types">
```

Comment:
`<h4 id="algorithm-get-types" algorithm dfn export>`

This defines an algorithm named "Which data types ought to be removed for |response|?". If you want a short name for internal linking, you should be able to add a `local-lt="get types"` attribute, but I'm not entirely sure that works in headers. Try it out! :)